wazuh/wazuh-dashboard-plugins
# Fix Switchtab exception in dashboards

### Description
Fix Switchtab exception in dashboards and some minor performance fixes to de Events and Kibana Discover module.

### Issues Resolved
#5900 

### Test
1. Navigate to threat hunting, refresh the page several times and check each time there are no "switchtab" exceptions in the console.
2. Make use of filters in any dashboard and check they work properly.
3. Go to Events tab and check there are no duplicated requests and filters work properly.

### Check List
